_authentication.md

```diff
@@ -156,4 +207,4 @@ curl https://api.uphold.com/v0/me \
 
 You can use Basic Authentication by providing your username or email and password combination.
 
-If OTP (One Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required`. Execute the command above again, this time passing your OTP verification code as a header, like so: `OTP-Token: <OTP-Token>`.
\ No newline at end of file
+If OTP (One Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required`. Execute the command above again, this time passing your OTP verification code as a header, like so: `OTP-Token: <OTP-Token>`.
```

Comment:
Not sure what changed here. I've been getting this line marked as changed every time I've edited this file. Probably a different line ending or something like that. I've finally decided to include this change, to see if the problem gets fixed once and for all.

Reply:
> Probably a different line ending or something like that.

That's exactly it. On master this file does not have a newline at the end, and on this branch it does.

Many editors now do this by default, and it it a recommended practice. If you've manually removed this change from the previous commits it's likely that your editor has been trying to add the newline and you haven't let it :P So merging this change should indeed stop the issue from resurfacing, unless you (or someone else) uses an editor configured to _remove_ the final newline to edit this file (I am not aware of any editor doing that by default, though).